shacl/Av.ttl

```diff
@@ -74,15 +81,15 @@ recordtypes:Av
   sh:property [
     rdfs:label "Extra informatie"@nl ;
     sh:maxCount 1 ;
-    sh:path av:extraInformation ;
+    sh:path rico:descriptiveNote ;
     sh:order 12.0 ;
     sh:group av:contentDescriptionGroup ;
     sh:datatype xsd:string ;
     dash:editor  dash:TextAreaEditor ;
   ] ;
   sh:property [
     rdfs:label "Afgebeelde personen"@nl ;
```

Comment:
Met zowel een organisatie toegestaan als een persoon is het rdfs:label "Afgebeelde persoon" misschien wat te beperkend